tutorials/optimal_taxation_theory_and_simulation.ipynb

```diff
@@ -1096,7 +1113,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
```

Comment:
this is unavoidable?

Reply:
Yeah, I think this comes from my local python version using which I opened the notebook. But does not show up in the notebook view itself. I could change these to 3.6 on a text editor, and that shouldn't break anything. What do you think?